DeepCat.js

```diff
@@ -8,7 +8,8 @@
 ( function( $, mw ) {
 	var DeepCat = {},
 		keyString = 'deepcat:',
-		maxDepth = 15,
+		defaultDepth = 15,
+		maxDepth = 20,
```

Comment:
@Nudin, I'm so sorry. I'm not sure to which specific comment you are responding to, but I hope this is a huge misunderstanding. Are you in the office some day soon? I would love to help resolving this situation.